commands/github-post-standup.js

```diff
@@ -0,0 +1,62 @@
+const fs = require("fs");
+
+module.exports = {
+    name: "github-post-standup",
+    description: "Post a comment on a GitHub discussion",
+    execute(command, message, args) {
+        // -github-post-standup: Posts desired comment on a GitHub Discussion
+        if (command === "github-post-standup") {
+            postComment(args);
+        }
+
+        function readToken() {
+            try {
+                const data = fs.readFileSync("./github-token.txt", "utf8");
+                return data;
+            } catch (err) {
+                console.error(err);
+                return message.reply(
+                    "Your GitHub Personal Access Token could not been read. Please set it again using -github-info."
+                );
+            }
+        }
+
+        // Post Comment function
+        async function postComment(args) {
+            // GitHub Variables
+            let githubToken = readToken();
+
+            // Intialise GitHub API
+            const { Octokit } = require("@octokit/core");
+            const { restEndpointMethods } = require("@octokit/plugin-rest-endpoint-methods");
```

Comment:
Best practice is to include these imports at the top of your file, unless you have a specific reason for including them in the function